etc/config-template.toml

```diff
@@ -27,4 +27,51 @@ prefix = "tikv"
 # set cluster id, must greater than 0.
 #cluster-id = 1
 # set pd address, host:port
-#pd = 
\ No newline at end of file
+#pd = 
+
+# For detailed explanation please refer to https://github.com/facebook/rocksdb/blob/master/include/rocksdb/options.h
+[rocksdb]
+# compression method (if any) is used to compress a block.
+#   kNoCompression
```

Comment:
The official options are kNoCompression, kSnappyCompression ... but it is ok for me to simplify those options if you insist.